src/OidcClient.php

```diff
@@ -387,7 +409,11 @@ private function requestTokens(
     string $grantType,
     ?string $code = null,
     ?string $redirectUrl = null,
-    ?string $refreshToken = null): OidcTokens
+    ?string $refreshToken = null,
+    ?string $subjectToken = null,
+    ?string $requestedTokenType = null,
```

Comment:
I don't think the `$requestedTokenType` is used anywhere?

Reply:
Done, I removed the `$requestedTokenType` arg.